docs/application/dotnet/guides/app-management/overview.md

```diff
@@ -24,6 +24,9 @@ You can use the following application management features in your .NET applicati
 
   You can retrieve information about the components installed on the device. You can also get information about the currently running components.
 
+- [Component Port](component-port.md)
+  The component can communicate with other components using component ports. You can send and receive requests using the serializable data format.
```

Comment:
@hjhun 

There needs to be a line space between 27 & 28.

Reply:
Done

Reply:
```suggestion
The components can intercommunicate using component ports. You can send and receive requests using the serializable data format.

Reply:
Done